simulator/runnergroup_visibility.go

```diff
@@ -18,30 +18,47 @@ func (c *Simulator) GetRunnerGroupsVisibleToRepository(ctx context.Context, org,
 		panic(fmt.Sprintf("BUG: owner should not be empty in this context. repo=%v", repo))
 	}
 
-	runnerGroups, err := c.Client.ListOrganizationRunnerGroups(ctx, org)
-	if err != nil {
-		return visible, err
-	}
+	if c.Client.GithubBaseURL == "https://github.com/" {
```

Comment:
Is there any better way to detect whether we configure with GitHub Enterprise server?

Reply:
Can I feature flag these changes in case they break any customers?